packages/contextual-help/test/contextual-help.test.ts

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { elementUpdated, expect, fixture, oneEvent } from '@open-wc/testing';
+import { nextFrame } from '@spectrum-web-components/overlay/src/AbstractOverlay.js';
+import { sendKeys } from '@web/test-runner-commands';
+import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+import { ContextualHelp } from '../src/ContextualHelp.js';
+import { ContextualHelpMarkup } from '../stories';
+
+describe('ContextualHelp', () => {
+    testForLitDevWarnings(
+        async () => await fixture<ContextualHelp>(ContextualHelpMarkup())
+    );
+    it('loads default contextual-help accessibly', async () => {
+        const el = await fixture<ContextualHelp>(ContextualHelpMarkup());
+
+        await elementUpdated(el);
+
+        await expect(el).to.be.accessible();
+
+        const button = document
+            .querySelector('sp-contextual-help')
+            ?.shadowRoot?.querySelector('sp-action-button');
+        expect(button).to.have.attribute('aria-label', 'Informations');
+
+        el.variant = 'help';
+
+        await elementUpdated(el);
+        expect(button).to.have.attribute('aria-label', 'Help');
+    });
+    it('is a popover on web', async () => {
+        const el = await fixture<ContextualHelp>(ContextualHelpMarkup());
+
+        await elementUpdated(el);
+        const trigger = el.shadowRoot?.querySelector('#trigger') as HTMLElement;
+
+        let popover = el.shadowRoot?.querySelector('sp-popover');
+        expect(popover).not.to.exist;
+
+        const opened = oneEvent(el, 'sp-opened');
+        trigger.click();
+        await opened;
+
+        popover = el.shadowRoot?.querySelector('sp-popover');
+        expect(popover).to.exist;
+        const headingSlot = popover?.querySelector(
+            'slot[name="heading"]'
+        ) as HTMLSlotElement;
+        const heading = headingSlot.assignedElements()[0].textContent;
+        expect(heading).to.equal('Permission required');
+
+        const closed = oneEvent(el, 'sp-closed');
+        await sendKeys({
+            press: 'Escape',
+        });
+        await closed;
+        await nextFrame();
+        await nextFrame();
```

Comment:
Why do we need to wait for a few frames here?

Reply:
Waiting for DOM clean up to complete, similar to [here](https://github.com/adobe/spectrum-web-components/blob/7a86d8c449e4cd8c8b94992139aa09a7656dd071/packages/overlay/test/overlay-directive.test.ts#L194). The test fails without it because it still finds the popover in the DOM.